CONTRIBUTING.md

```diff
@@ -1,6 +1,23 @@
 Contribute to Che-Theia
 ================
 
+Table of content
+- [Introduction](#introduction)
+- [Devfiles](#devfiles)
```

Comment:
Could you rename the Devfile for contributing to Theia or Che-Theia extensions or Che-plugins to Contribute to Theia or Che-Theia extensions or Che-plugins and add that item to the TOC ?